src/common/telemetry/src/logging.rs

```diff
@@ -68,6 +68,9 @@ pub struct LoggingOptions {
 
     /// The logging options of slow query.
     pub slow_query: SlowQueryOptions,
+
+    /// The maximum number of log files set by default.
+    pub max_log_files: usize,
```

Comment:
You may need to define max_log_files before slow_query to avoid toml deserialization error. I remember that we have a unit test for it so we can check the CI later.

Reply:
Got it. I am now putting it right after `log_format`.